pwa/static/themes/rotterdam.css

```diff
@@ -117,9 +117,9 @@
 
   --web-app-color-background-page-top-bar: var(--web-app-color-white);
 
-  --web-app-color-background-footer: #eff4f6;
-  --web-app-color-footer-headings: auto;
-  --web-app-color-footer-links: var(--web-app-color-black);
+  --web-app-color-background-footer: #00811f;
```

Comment:
Gebruik in plaats van de hex kleur liever `var(--rotterdam-color-base-green)`.

Je kunt hier de color tokens opzoeken: https://nl-design-system.github.io/themes/?path=/story/rotterdam-color--page

Dit geldt eigenlijk ook voor alle andere kleuren in `.rotterdam-theme`, zoals `var(--web-app-color-white)`, daar kun je beter `var(--rotterdam-color-base-white)` gebruiken.